tests/load/pipeline/test_csv_loading.py

```diff
@@ -0,0 +1,73 @@
+import os
+import pytest
+
+from dlt.common.utils import uniq_id
+
+from tests.load.pipeline.utils import destinations_configs, DestinationTestConfiguration
+from tests.cases import arrow_table_all_data_types, prepare_shuffled_tables
+from tests.pipeline.utils import assert_data_table_counts, assert_load_info, load_tables_to_dicts
+from tests.utils import TestDataItemFormat
+
+
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(default_sql_configs=True, subset=["postgres", "snowflake"]),
+    ids=lambda x: x.name,
+)
+@pytest.mark.parametrize("item_type", ["object", "table"])
+def test_load_csv(
+    destination_config: DestinationTestConfiguration, item_type: TestDataItemFormat
+) -> None:
+    os.environ["DATA_WRITER__DISABLE_COMPRESSION"] = "True"
```

Comment:
note to self: if I set up those helper functions correctly, we can also easily inspect gzipped files without this setting. ibis / dataframes will make this stuff obsolete anyway because we can use that in the tests in the future

Reply:
@sh-rp I did it already for tead_text is fsclient. will push changes thoday